@uportal/user-profile-menu/src/App.vue

```diff
@@ -45,6 +45,14 @@ export default {
 </script>
 
 <style>
+
+:root {
+  --user-profile-dropdown-item-bg-color: black;
+  --user-profile-dropdown-item-fg-color: white;
+  --user-profile-dropdown-item-bg-color-hover: white;
+  --user-profile-dropdown-item-fg-color-hover: black;
```

Comment:
minor preference for `foreground` and `background` over `fg` and `bg`, but not a blocker